src/ParamPassing.hh

```diff
@@ -44,14 +44,14 @@ namespace sdf
     /// \param[in] _elemName The element name, such as "model", "link",
     /// "collision", "visual".
     /// \param[in] _elemId The element identifier
-    /// \param[in] _prefixModelName True to add prefix of model name when
-    /// looking for element
+    /// \param[in] _isParentElement Is true if _elemId is the parent. Is true
```

Comment:
Can you clarify the second sentence? I'm not sure what `Is true and only use for the add action` indicates.

Reply:
Sorry, does this help clarify? 5f648cc55506a3604237e533a7385cd43d8aa34f

When `getElementById` is used, it uses `_elemName` and `_elemId` to verify the correct element is found for all actions except for `add`. For an `add` action, only the `_elemId` is used and specifies the parent element. For example, 

```xml
<model name="A">
  <link name="B"/>
</model>
```

and under `//experimental:params` the user would like to add:

```xml
<visual element_id="A::B::new_visual" action="add">
  ...
</visual>
```

Then `_elemId="A::B"` and `_isParentElement=true`

Reply:
Thanks, that helps.